packages/framer-motion/src/components/AnimatePresence/__tests__/AnimatePresence.test.tsx

```diff
@@ -445,7 +446,7 @@ describe("AnimatePresence", () => {
     })
 
     test("Fast animations with wait render the child content correctly (strict mode disabled)", async () => {
-        const promise = new Promise<boolean>((resolve) => {
+        await new Promise<boolean>((resolve) => {
```

Comment:
In theory if this reslove with false, we would not catch this now.
If it rejects, than the await throws and that will fail the test, right?